cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/JsonUtil.java

```diff
@@ -73,13 +75,34 @@
     }
 
     public static String convertToJson(Object value) {
-        if (mapper.canSerialize(value.getClass())) {
+        return convertToJsonString(value);
+    }
+
+    public static String convertToJson(Object value, boolean useIdentation) {
+        if (useIdentation) {
+            return convertToJsonUsingFeatures(value, Feature.INDENT_OUTPUT);
+        }
+
+        return convertToJsonUsingFeatures(value);
+    }
+
+    private static String convertToJsonUsingFeatures(Object value, Feature... features) {
+        mapper.enable(features);
+        String converted = convertToJsonString(value);
+        mapper.disable(features);
+        return converted;
+
+    }
+
+    private static String convertToJsonString(Object value) {
+        if (Objects.isNull(value) || mapper.canSerialize(value.getClass())) {
```

Comment:
Why not just == null? Objects.isNull is intended to be used in streams as a method reference.